oslabs-beta/React-Labyrinth
# AL/tsconfig

## Overview

**Issue Type**


**Description**
Deleted unused images and added property in tsconfig to remove warnings and ignore node_modules/index.d.ts when running npx tsc

**Steps to Validate Feature**

1. Run `npx tsc`
2. Inspect the CLI
3. You will see that there are no warnings regarding index.d.ts when checking for node_modules

**Previous behavior**
We had warnings when running `npx tsc` about repeating node_modules/index.d.ts with testing

**Expected behavior**
Now, no warnings will appear.
